web/components/application/hacker/pages/PageThree/PageThree.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import { HackerApplicationPageTemplate } from '../PageTemplate';
+import { Page } from '../../../Page';
+import { constraints } from './PageThreeConstraints';
+import { Select } from '../../../../input/select';
+import { Checkbox, CheckboxGroup } from '../../../../input/buttons/CheckboxGroup';
+import { sources } from '../../../sources';
+
+// Emojis
+import robot from '../../../../../assets/emoji/robot.svg';
+import floppy from '../../../../../assets/emoji/floppy.svg';
+import briefcase from '../../../../../assets/emoji/briefcase.svg';
+
+class PageThree extends Page {
+  constructor(props) {
+    super(props);
+    this.constraints = constraints;
+  }
+
+  render() {
+    const {
+      application: {
+        source,
+        isCheckedPrivacyPolicy,
+        isCheckedCodeOfConduct,
+        isCheckedDataReporting,
+        isCheckedDocuments,
+      },
+    } = this.props;
+
+    return (
+      <HackerApplicationPageTemplate {...this.getPageTemplateProps()}>
+        <h2>Almost there...</h2>
+
+        <Select
+          options={sources}
+          value={source}
+          name="source"
+          label="How did you hear about nwHacks?"
+          placeholder="Select an option"
+          className="margin-ends-giga"
+          onChange={({ value: newSource }) => this.updateApplication({ source: newSource })}
+          onBlur={() => this.setFieldAsBlurred('source')}
+          error={this.getErrorIfBlurred('source')}
+          isSearchable
+          allowNewOption
+          />
+
+        <CheckboxGroup
+          name="MLH-checkbox"
+          label={<div><img className="vertical-align-top emoji" alt="🏅" src={robot}/> We participate in Major League Hacking (MLH) 
+            as a MLH Member Event. You authorize us to share certain application/registration information for event administration, 
+            ranking, MLH administration, and occasional messages about hackathons in line with 
+            the <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank">MLH Code of Conduct.</a></div>}
+          className="dir-row margin-ends-giga"
```

Comment:
if you want less space on the top of the checkboxes, you can do this:

`"dir-row margin-top-l margin-bottom-giga"`.

Reply:
I tried this but it actually made the margins bigger?

Reply:
ok nvm then. i don't think it's a big deal

Reply:
I fixed the spacing in one of the later commits